tools/aws_lab_setup/provision_lab.yml

```diff
@@ -2,7 +2,9 @@
   hosts: localhost
   connection: local
   become: no
-  gather_facts: no
```

Comment:
Maybe I'm missing something. Do we need to gather facts for the localhost connection? Seems to me we can still keep `gather_facts: no`.

Reply:
originally when I install ssh private key I was using the ansible facts... you are correct that we can probably turn it off since I had to change to a shell module to accomplish that task and didn't need to install the local user (it just assumed whoever you are running it from)